### PR TITLE
Fix updating pool settings while update validator stake

### PR DESCRIPTION
**Pull request by ramakser**
## Purpose
Commission rates are set to 100% in case they are not explicitly set in the validators transaction.  Prevent changing commission rates while updates validator stake

## Changes

Add ```private func loadPoolParameters(for bakerID: Int) {...}``` method to retrieve  validator pool commissionRates  in case if profile is `validator`, or load them from chain parameters like it was before. So in this case there will not be changes rates to 100%, while user change only validator stake.

### CODE REVIEW

**Review by bisgardo — APPROVED** [review]
It took a little while to understand this PR. I'm not too happy about the duplication this introduces and feel like it could have been done in a simpler way: For example you could have constructed the stream in such a way that the final result handled in `sink` included the appropriate `CommissionRates` value. Based on whether the account is a baker or not that object would have been created from the pool info or crypto params earlier in the pipeline (like `delegatedCapital` was).

But it appears to be correct as it is, so I'm approving and letting you decide.

File: ConcordiumWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved
[the remark this answers is not in the document]
Comment (bisgardo) [reply ConcordiumWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: Removing this file, that is.

[the remark this answers is not in the document]
Comment (ramakser) [reply ConcordiumWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: looks like Xcode removes this file every time when I commit changes 😩

**Comment by ramakser** [comment]
> It took a little while to understand this PR. I'm not too happy about the duplication this introduces and feel like it could have been done in a simpler way: For example you could have constructed the stream in such a way that the final result handled in `sink` included the appropriate `CommissionRates` value. Based on whether the account is a baker or not that object would have been created from the pool info or crypto params earlier in the pipeline (like `delegatedCapital` was).
> 
> But it appears to be correct as it is, so I'm approving and letting you decide.

updated ^

File: ConcordiumWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -1,149 +1,149 @@
...
+       "kind" : "remoteSourceControl",
+       "location" : "https://github.com/WalletConnect/WalletConnectSwiftV2.git",
+       "state" : {
+         "revision" : "84b09a13b48d08f1092a7494e2ad70cbb92545c2",
+         "version" : "1.6.11"
+       }
+     }
+   ],
+   "version" : 2
+ }
```
Comment (bisgardo) [inline ConcordiumWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:149]: 🤔

**Review by bisgardo — APPROVED** [review]
I think this would have been nicer if done using async/await rather than these streams. I realize you're just following the existing code style, so need to change it. Just wanted to point it out :)

File: ConcordiumWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (ramakser) [reply ConcordiumWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:149]: will do

**Pull request merged**